scripts/firetext.js

```diff
@@ -92,6 +89,19 @@ firetext.init = function () {
 			firetext.isInitialized = true;
 		});	
 	});
+	
+	navigator.mozL10n.ready(function () {
```

Comment:
For symmetry with the call to `once`, more or less. We could also make two listeners, if you want the bugsense thing in language.js.